common/v2/services/UnstoppableService/index.ts

```diff
@@ -0,0 +1,6 @@
+import UnstoppableResolution from './unstoppableResolution';
+
+export { getResolvedAddress } from './functions';
+export { isValidDomain } from './validators';
+
+export default UnstoppableResolution;
```

Comment:
I don't think you need so many services. All the job can be done by `UnstoppableResolution`. Others do not do much on top of it.